src/tracer.js

```diff
@@ -54,11 +55,13 @@ export default class Tracer {
      *        as process-level tags on the Tracer itself.
      * @param {Object} [options.metrics] - instance of the Metrics class from ./metrics/metrics.js.
      * @param {Object} [options.logger] - a logger matching NullLogger API from ./logger.js.
+     * @param {Boolean} [upsampling] - determines whether to resample child spans of unsampled parents.
      */
     constructor(serviceName: string,
             reporter: Reporter = new NoopReporter(),
             sampler: Sampler = new ConstSampler(false),
-            options: any = {}) {
+            options: any = {},
+            upsampling: any = {}) {
```

Comment:
> because new argument creates an unnecessary API change (many tests are affected for no reason)

The test changes were leftover from a previous change, they are unnecessary. 

I'm not very clear on why something goes into `options` vs using function parameters as my js experience is limited. Could you elucidate?